### ISSUE TITLE
Breaks with ember 2.9.0

### ISSUE THREAD

**Issue by jacobq** [ember-animation/liquid-fire#510]
It looks like `liquid-fire` caused my app to break when I switched to ember 2.9.0. The error was
`Uncaught Error: Assertion Failed: A helper named "-get-dynamic-var" could not be found`
I just downgraded back to 2.8.2 for now but thought you might want to know in case this is fixable here. (I didn't dig in this time.)

**Comment by abrahamspaa** [comment]
After I have upgraded to 2.9.0 It throws an error on 

https://github.com/ember-animation/liquid-fire/blob/master/version-specific-2.9/index.js#L3
```
(anonymous function) @ VM2335:1containingElement @ index.js:11_transition @ liquid-versions.js:73childDidRender @ liquid-versions.js:119send @ ember.debug.js:41869superWrapper @ ember.debug.js:24801triggerAction @ ember.debug.js:37339sendAction @ ember.debug.js:41853(anonymous function) @ liquid-child.js:22tryCatch @ ember.debug.js:50950invokeCallback @ ember.debug.js:50965publish @ ember.debug.js:50933(anonymous function) @ ember.debug.js:40413invoke @ ember.debug.js:990flush @ ember.debug.js:1054flush @ ember.debug.js:862end @ ember.debug.js:176run @ ember.debug.js:298join @ ember.debug.js:318run.join @ ember.debug.js:23767(anonymous function) @ ember.debug.js:23830dispatch @ jquery.js:4670elemData.handle @ jquery.js:4338trigger @ jquery.js:4579(anonymous function) @ jquery.js:5289each @ jquery.js:384each @ jquery.js:136trigger @ jquery.js:5288triggerChange @ select2.js:1168onSelect @ select2.js:2465selectHighlighted @ select2.js:1870(anonymous function) @ select2.js:807(anonymous function) @ select2.js:681dispatch @ jquery.js:4670elemData.handle @ jquery.js:4338
jquery.js:4670 TypeError: getViewBounds is not a function
    at containingElement (index.js:11)
    at Class._transition (liquid-versions.js:73)
    at Class.childDidRender (liquid-versions.js:119)
    at Class.send (ember.debug.js:41869)
    at Class.superWrapper [as send] (ember.debug.js:24801)
    at Class.triggerAction (ember.debug.js:37339)
    at Class.sendAction (ember.debug.js:41853)
    at liquid-child.js:22
    at tryCatch (ember.debug.js:50950)
    at invokeCallback (ember.debug.js:50965)
```

**Comment by ef4** [comment]
Yes, ember 2.9 didn't end up shipping with the same features that were in
2.9 beta. We need to change the version check.

**Comment by ef4** [comment]
Fixed in d860d2271b019210f6e5aef5837321ff9fdb8b7b and released as 0.26.2.
